src/operator/tensor/elemwise_binary_op.h

```diff
@@ -305,6 +316,59 @@ class ElemwiseBinaryOp : public OpBase {
     return dispatched;
   }
 
+
+  /*!
+   * \brief Allow one of the inputs to be dense and produce a dense output
```

Comment:
Add a comment that for rsp, it only support rsp, rsp -> rsp

Reply:
Done